### PR TITLE
Assignments 07.05.24 IF Statements

### PR DESCRIPTION
**Pull request by CynthiaWahome**
Assignment on chapter_02
1. life_stages.py
2. favorite_fruit.py
3. ordinal_numbers.py

Note to Self:
I wanted this branch to only have assignments. How can I remove the other folders

### CODE REVIEW

File: chapter_3/favourite_fruit.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+if fruit == favorite_fruits[0]:
+    print("I really like",fruit,"!")
+if fruit == favorite_fruits[2]:
+    print("I really like",fruit,"!")
+if fruit == favorite_fruits[3]:
+    print("I really like",fruit,"!")
+
+else:
+    print("Iza joh! That fruit is not in the list")
```
Comment (ProgramKingAbel) [inline chapter_3/favourite_fruit.py:12]: Good job implementing the if statements:

- [ ] Remove the else block as it is only tied to the last if statement

File: chapter_3/ordinal_numbers.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+for num in nums:
+    # print(nums)
+    if num == 1:
+        print(num,"st")
+    elif num == 2:
+        print(num,"nd")
+    elif num == 3:
+        print(num,"rd")
+    elif num < 4 or num < 10:
+        print(num,"th")
```
Comment (ProgramKingAbel) [inline chapter_3/ordinal_numbers.py:11]: syntax is okay but for the logic, consider using the following in the last statement:

`elif num > 3 or num < 10`

for your condition

**Review by ProgramKingAbel — CHANGES_REQUESTED** [review]
#1 Good Job
Job well done ... You have a strong grasp of the if conditional statements. 

- [ ] You cannot have a branch solely for assignments ... a specific folder dedicated to that within the main branch would do. You checkout and do the tests on the assignments folder for each chapter.

**Comment by CynthiaWahome** [comment]
Thanks @ProgramKingAbel 
These changes have been implemented

**Pull request merged**